code/game/machinery/computer/pod.dm

```diff
@@ -6,7 +6,7 @@
 	icon_screen = "command"
 	light_color = "#00b000"
 	circuit = /obj/item/circuitboard/pod
-	var/id = 1.0
+	id = 1.0
```

Comment:
wat. Remove?

Reply:
these are used by pods internally

Reply:
For comparison. Thus, you need something (a mapper) to set them. Ergo, the default value of null is sufficient and this can be removed.